lmdeploy/vl/model/yi.py

```diff
@@ -80,11 +82,28 @@ def init_yi_model():
         yield
 
 
+@VISION_MODELS.register_module()
 class YiVisionModel(LlavaVisionModel):
     """Yi visual model."""
 
-    def __init__(self, model_path, with_llm: bool = False):
-        super().__init__(model_path=model_path, with_llm=with_llm)
+    def __init__(self,
+                 model_path: str,
+                 with_llm: bool = False,
+                 max_memory: Dict[int, int] = None,
+                 **kwargs):
+        super().__init__(model_path=model_path,
+                         with_llm=with_llm,
+                         max_memory=max_memory)
```

Comment:
kwargs may need to pass to super init since its father class is `LlavaVisionModel`